scout/server/extensions/mongo_extension.py

```diff
@@ -9,14 +11,14 @@ class MongoDB:
     def init_app(app):
         """Initialize from flask"""
 
-        db_name = app.config.get("MONGO_DBNAME", "scout")
+        db_name = os.environ.get("MONGO_DBNAME") or app.config.get("MONGO_DBNAME", "scout")
```

Comment:
Ah, ok, you don't want to pass the yaml? Or why do you want to set these from ENV? 😊

Reply:
I wasnt't thinking about the yaml file. In prod settings (or even development settings) you don't need to set up these env vars since you are passing the path to the config file. But you might need them in the eventuality you can't change the demo config file but still want to customize the database you are connecting to (like in out demo docker-compose setup)

Reply:
Because otherwise in the demo config file the database host is localhost (not mongodb), port 27017 (which is fine), and database name is scout (not scout-stage)

Reply:
Well, yes, it is a demo so all good. For a prod solution, we would of course really just like to skip the yaml. And as a fallback option pass it. But that could be done with like mounting over the default file location when firing up the instance, so not really a big issue, right?

Reply:
I don't think we should keep using the yaml file, especially in prod. 

When launching the cli, we could fix it that we pass database uri, database name and OMIM api key as env vars instead. As an alternative we might add the OMIM api key as one of the params of the .py config file (of course we should make sure that the app reads it and uses it.

So we can deprecate the use of the yaml file for good! 😀